tools/gbench/report.py

```diff
@@ -302,7 +310,7 @@ def test_utest(self):
             ['BM_One', '-0.1000', '+0.1000', '10', '9', '100', '110'],
             ['BM_Two', '+0.1111', '-0.0111', '9', '10', '90', '89'],
             ['BM_Two', '+0.2500', '+0.1125', '8', '10', '80', '89'],
-            ['U-test', '(p-value)', '0.2207', '0.6831'],
+            ['U-test', '(p-value)', '0.2207', '0.6831', 'Repetitions:', '2.', 'WARNING:', 'Results', 'unreliable!', '9+', 'repetitions', 'recommended.'],
```

Comment:
a comment on output: should it have the bm name in the 'u-test' bit so people running the output through grep/ag/whatever will catch the u-test for the bm they want?

Reply:
Ah, good idea..
How do you want it spelled, `_utest_pvalue` ?

Reply:
or just `BM_Two_p_value` ?

Reply:
But now that i have actually looked, having such a long suffix as `_utest_pvalue`, causes
the need to realign the columns (since the first column may now be wider).
So `_utest_pvalue` is /better/, but more troublesome.

Reply:
So i have thought, and ok, let's just use `_pvalue`.
Avoids the headache with column width readjusting _for now_.

Reply:
Pushed the `_pvalue` variant.